config

```diff
@@ -190,7 +196,13 @@ bindsym $mod+Ctrl+Shift+Down move workspace to output down
 bindsym $mod+Ctrl+Shift+Up move workspace to output up
 bindsym $mod+Ctrl+Shift+Right move workspace to output right
 
-## Modify // Next Window Orientation // <> Backspace ##
+## Modify // Vertical Window Orientation // <> v ##
+bindsym $mod+v split vertical
+
+## Modify // Horizontal Window Orientation // <> g ##
+bindsym $mod+g split horizontal
```

Comment:
All of this looks great @hypergig , but out of curiosity, why did you choose `g` here?

Reply:
Honestly, I usually chose `b` because it is next to `v` (and in my head, I mnemonically connect `b` to 'butt up against`) but `b` is already used for bluetooth and nothing worse than when they change a shortcut on you that you have been using for a while. I figure if we ever did a big shortcut refactor, we can address it then. The f keys are popular for system shortcuts?

Reply:
Ah, got it.  Thanks for the info.  I have avoided using the Function keys for anything in Regolith.  No real reason, just didn't see the need to use them yet.

Reply:
I couldn't agree with you more!